src/transformers/modeling_tf_bert.py

```diff
@@ -471,54 +575,74 @@ def build(self, input_shape):
         super().build(input_shape)
 
     def call(self, hidden_states):
-        hidden_states = self.transform(hidden_states)
-        hidden_states = self.input_embeddings(hidden_states, mode="linear")
+        hidden_states = self.transform(hidden_states=hidden_states)
+        hidden_states = self.input_embeddings(input_ids=hidden_states, mode="linear")
         hidden_states = hidden_states + self.bias
 
         return hidden_states
 
 
-class TFBertMLMHead(tf.keras.layers.Layer):
+class TFBertOnlyMLMHead(tf.keras.layers.Layer):
```

Comment:
Is the renaming really necessary here? This is quite backwards breaking...(even if it's only the `TFBertMLMHead`)

Reply:
Not since the new method to load the TF weights :)